index.js

```diff
@@ -3700,6 +3700,52 @@
   S.pairs =
   def('pairs', {}, [$.StrMap(a), $.Array($.Pair($.String, a))], pairs);
 
+  //# insert :: String -> a -> StrMap a -> StrMap a
+  //.
+  //. Inserts a given key and value into a shallow clone of a given
+  //. string map. Overrides the value if a key already exists.
+  //.
+  //. Equivalent to Haskell's `insert` function. Similar to Clojure's `assoc`
+  //. function.
+  //.
+  //. ```javascript
+  //. > S.insert('c', 3, {a: 1, b: 2})
+  //. {a: 1, b: 2, c: 3}
+  //.
+  //. > S.insert('a', 4, {a: 1, b: 2})
+  //. {a: 4, b: 2}
+  //. ```
+  function insert(key, val, strMap) {
+    var result = Z.concat(strMap, {});
+    result[key] = val;
+    return result;
+  }
+  S.insert =
+  def('insert', {}, [$.String, a, $.StrMap(a), $.StrMap(a)], insert);
+
+  //# remove :: String -> StrMap a -> StrMap a
+  //.
+  //. Removes a given key and its value from a shallow clone of a given
+  //. string map. The original string map is returned if the key doesn't exist.
```

Comment:
The second sentence is no longer accurate.

Reply:
You are correct, I apologize for the oversight! Pushing an update now.